pages/interface/components/TabCoinCount/index.js

```diff
@@ -9,7 +9,7 @@ export default function TabCoinCount({ amount, direction, mode = 'tooltip', sx }
     },
     full: {
       iconSize: 20,
-      text: `${amount?.toLocaleString('pt-BR')} TabCoin${amount !== 1 && 's'}`,
+      text: `${amount?.toLocaleString('pt-BR')} TabCoin${amount === 1 ? '' : 's'}`,
```

Comment:
Podemos aproveitar para tratar o `-1` também.

```suggestion
      text: `${amount?.toLocaleString('pt-BR')} TabCoin${Math.abs(amount) === 1 ? '' : 's'}`,
```

Se `amount` for `undefined`, `Math.abs(amount)` retornará `NaN`. Se for `null`, retornará `0`, então continuará funcionando nesses cenários.

Reply:
Boa! Não tinha considerado o valor negativo.

Reply:
@smrsassa acabei de perceber um detalhe sobre sua mensagem de commit e editei meu comentário principal, mas vou colocar aqui também porque não sei se você viu a edição:

> Aproveitando que sugeri uma edição, depois você pode realizar um squash dos commits, deixando a mensagem de commit em inglês, por favor?

Reply:
Claro, pode deixar